### PR TITLE
Support all the 0.2 Ferris variants

### PR DESCRIPTION
**Pull request by pierrechevalier83**
For the Compact, the High and the Mini, create a new directory so each
keyboard may have the correct USB descriptor and a readme with more
specific information about it.

For the Bling, also add support for the underglow functionality.

Change the "MANUFACTURER" string to "Cuddly Keyboards Ltd.", the
name of the company I incorporated to sell keyboards, and change the
default descriptor from "Ferris the keeb" to "Ferris 0.2" which is more
descriptive.
I didn't update the 0.1 variants as I don't intend to sell these kits
with "Cuddly Keyboards". The firmware is mostly there to support
existing users.

Update the "hardware availability" to point to my new website:
cuddlykeyboards.com.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ferris/0_2/bling/bling.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }, {
+        // LED Index to Physical Position
+        // |      Left        ||    Right      |
+        // |    | 1  |   | 2  || 3 |   | 4 |   |
+        // | 0  |    |   |    ||   |   |   | 5 |
+        // |    |    |   |    ||   |   |   |   |
+        // |    |    |   | 10 || 9 |   |   |   |
+        // |    | 12 |   |    ||   |   | 7 |   |
+        // | 13 |    |   | 11 || 8 |   |   | 6 |
+        P(1, 0), P(0, 1), P(0, 3), P(0, 4), P(0, 6), P(1, 7),
```
Comment (drashna) [inline keyboards/ferris/0_2/bling/bling.c:71]: Could you remove the macros, and use explicit coordinates?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ferris/0_2/bling/bling.c (hunk 1 above)
Comment (pierrechevalier83) [reply keyboards/ferris/0_2/bling/bling.c]: Thanks for the review, @drashna. I've done this now, and on the way spotted and fixed a bug where the order of the 4 central, bottom LEDs was incorrect. The animations now look better :)

**Review by drashna — APPROVED** [review]

File: keyboards/ferris/0_2/readme.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 # Ferris 0.2
 
-![Ferris 0.2 - bling, top view](https://i.imgur.com/LwKlmnzh.jpg)
-![Ferris 0.2 - bling, bottom view](https://i.imgur.com/qGnYGVOh.jpg)
+![Ferris 0.2 family](https://i.imgur.com/SDVOlyZ.jpg)
 
 An stm32f072 based split 34 keys column staggered keyboard named and decorated after the rustlang mascott. All PCB files and some thoughts on the design are available on the [project's github page](https://github.com/pierrechevalier83/ferris)
```
Comment (noroadsleft) [inline keyboards/ferris/0_2/readme.md:5]: Some wordsmithing.

```suggestion
An STM32F072-based split 34-key column-staggered keyboard named and decorated after the Rust mascot. All PCB files and some thoughts on the design are available on the [project's github page](https://github.com/pierrechevalier83/ferris).
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Just an edit on the readme file.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**